submissions/AnnaStoyano/js-dom/script.js

```diff
@@ -0,0 +1,106 @@
+let download = function () {
+    let promise = fetch('./information.json');
+    promise.then(res => res.json())
+        .then(data => {
+            addMenuItems(data.faculties);
+            addContentItems(data.faculties);
+            return data;
+        })
+        .then(data=>{
+            menuHandler(data);
+            menuIconHandler();
+        })
+        
+}
+
+download();
+
+let addMenuItems = function (arr) {
```

Comment:
Let's use `let` and `const` in appropriate cases. More info about it [here](https://www.freecodecamp.org/news/var-let-and-const-whats-the-difference/)